src/sdl2.nim

```diff
@@ -394,52 +394,52 @@ const
 #      (SDL_static_cast(Uint32, SDL_static_cast(Uint8, (B))) << 8) | \
 #      (SDL_static_cast(Uint32, SDL_static_cast(Uint8, (C))) << 16) | \
 #      (SDL_static_cast(Uint32, SDL_static_cast(Uint8, (D))) << 24))
-template SDL_FOURCC (a,b,c,d): expr =
-  a or (b shl 8) or (c shl 16) or (d shl 24)
+template SDL_FOURCC (a,b,c,d: uint8): uint32 =
+  uint32(a) or (uint32(b) shl 8) or (uint32(c) shl 16) or (uint32(d) shl 24)
 
-template SDL_DEFINE_PIXELFOURCC*(A, B, C, D: expr): expr =
-  cint(SDL_FOURCC(A.int, B.int, C.int, D.int))
+template SDL_DEFINE_PIXELFOURCC*(A, B, C, D: char): uint32 =

```

Comment:
well actually that is exactly what it is supposed to be:
https://github.com/nim-lang/sdl2/blob/master/src/sdl2.nim#L508